eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -89,8 +89,6 @@ function Retry([scriptblock] $Action, [int] $Attempts = 5) {
             return $Action.Invoke()
         } catch {
             if ($attempt -lt $Attempts) {
-                $sleep *= 2
```

Comment:
Do you want to also remove this in the resource deployment scripts?

Reply:
Possibly. It looks like it's mostly wrapping `New-*` type calls, which we probably don't want backoff for anyway. I can do some testing.

Reply:
Also a test run for this PR took 30 minutes with a connection failure on china cloud, so I have to do some further digging.